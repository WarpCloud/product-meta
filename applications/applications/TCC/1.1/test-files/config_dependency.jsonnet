{
  application_type: "TCC",
  application_name: "tcc1",
  application_version: "1.1"
}
