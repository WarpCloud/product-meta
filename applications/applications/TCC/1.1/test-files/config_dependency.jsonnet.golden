{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "tcc1",
            "application_type": "TCC",
            "use_gntenant": "true"
         },
         "configs": {
            "tcc_cpu_limit": 1,
            "tcc_cpu_request": 0.10000000000000001,
            "tcc_memory_limit": 4,
            "tcc_memory_request": 1
         },
         "dependencies": [
            {
               "moduleName": "txsql",
               "name": "txsql"
            },
            {
               "moduleName": "ticket",
               "name": "ticket"
            },
            {
               "moduleName": "guardian-cas",
               "name": "guardian-cas"
            },
            {
               "moduleName": "gn-tenant",
               "name": "gn-tenant"
            }
         ],
         "instance_settings": { },
         "moduleName": "tcc",
         "name": "tcc1-tcc",
         "version": "1.1"
      }
   ],
   "instance_settings": { },
   "version": ""
}
